documentation/onboarding.md

```diff
@@ -16,7 +16,67 @@ Please read the [ARIA WG Process Document](process.md)
 * add **Channels**, "#aria"
 
 ### IRC command cheat sheet
-* [ARIA Issue 1404](https://github.com/w3c/aria/issues/1401#issuecomment-786384070)
+When you first enter the channel, type:
+`present+`
+
+Commands anyone can use:
+```
+q+  (to add yourself to the speaker queue)
+q+ to say [what you want to say]  (to add yourself to the speaker queue with reminder message)
+ack me  (take me off the q; or can use q-)
```

Comment:
q- and ack are not the same in the minutes especially if you used q+ to say version. You should use ack if you are acknowledged and q- if you just want to not speak any more